kani-driver/src/call_single_file.rs

```diff
@@ -122,6 +122,8 @@ impl KaniSession {
                 "symbol-mangling-version=v0",
                 "-Z",
                 "panic_abort_tests=yes",
+                "-Z",
+                "sanitizer=address",
```

Comment:
Can you please make sure the code from the standard library has these statements?

If not, we may have to add the same argument here: https://github.com/model-checking/kani/blob/ea710b36/tools/build-kani/src/sysroot.rs#L89

Reply:
Per our offline discussion, we will follow up on this separately.